GL-README.md

```diff
@@ -1,20 +1,28 @@
-# Groundlight' Monitoring Notification Server (MNS) as a Raspberry Pi Appliance
+# Groundlight Pi-Gen: OS images for Raspberry PI with Groundlight Tools
 
-This repo builds the OS image for a Groundlight MNS appliance running on Raspberry Pi hardware.
+This repo builds OS images for Groundlight tools and applications, including the bare python SDK, 
+and the Monitoring Notification Server (MNS).  The OS images are available in the [releases](https://github.com/groundlight/groundlight-pi-gen/releases) and can be installed with [Raspberry Pi imager](https://www.raspberrypi.com/software/).
 
-Use this repo to build the `.img` file which gets burned to an SD card.  When that SD card is booted on the Raspberry Pi, 
-it runs the [Groundlight Monitoring Notification Server](https://github.com/groundlight/monitoring-notification-server).
-The MNS provides a simple GUI to configure a Groundlight detector, grab images from it, and send notifications when
+There are several different images available, depending on your needs, from smallest to largest:
+
+- **`sdk`** - A minimal image with just the Python SDK installed.  This is the smallest image, and is suitable for running the SDK on a Raspberry Pi Zero W.  It is also suitable for running the SDK on a Raspberry Pi 3 or 4, if you don't need the GUI.
+- **`mns`** - an image with the [Groundlight Monitoring Notification Server (MNS)](https://github.com/groundlight/monitoring-notification-server) installed for headless use.  "Headless" means as server, without a GUI.  The MNS provides a simple GUI to configure a Groundlight detector, grab images from it, and send notifications when
```

Comment:
`"Headless" means as server, without a GUI. The MNS provides a simple GUI` -- I'm a little confused by that statement. Does this mean you can open an interface on a web browser pointed at the raspberry pi, but can't use this with a raspberry pi's screen?

Reply:
That's exactly what it means.  It has a server which serves the HTML, but you need to run the browser somewhere else.  I'll try to write that more clearly.